server/api/beverages.js

```diff
@@ -0,0 +1,78 @@
+const express = require('express');
+const router = express.Router();
+const { Beverage } = require('../db/beverage');
+
+router.get('/', async(req, res, next) => {
+  try {
+    const beverages = await Beverage.findAll();
+    res.send(beverages);
+  }
+  catch (err) {
+    next(err);
+  }
+});
+
+
+router.get('/:id',  async(req, res, next) => {
+  try {
+    const beverage = await Beverage.findByPK(req.params.id)
+    res.send(beverage);
+  }
+  catch (err) {
+    next(err);
+  }
+})
+
+router.post('/', async(req, res, next) => {
+  //allows only Admin to create new Beverages
+  if (req.isAdmin) {
+    try {
+      const newBeverage = await Beverage.create(req.body);
+      res.status(201).send(newBeverage);
+    }
+    catch (err) {
+      next(err);
+    }
+  }
+  else {res.status(401).send('Please login as an Administrator')}
+  //Redirect to /login route once it's set up
```

Comment:
I don't think we want to redirect on the backend. The /login route expects a req.body payload, and redirecting will not send anything. So it will always fail. I think sending a message with 401 is fine.

Reply:
Got it!